GameServer/Logic/GameObjects/Unit.cs

```diff
@@ -339,7 +339,7 @@ public override void onCollision(GameObject collider)
         */
         public virtual void autoAttackHit(Unit target)
         {
-            float damage = (_isNextAutoCrit) ? stats.getCritDamagePct() * stats.AttackDamage.Total : stats.AttackDamage.Total;
+            float damage = _isNextAutoCrit ? stats.getCritDamagePct() * stats.AttackDamage.Total : stats.AttackDamage.Total;
```

Comment:
I'd add brackets here for extra clarity
```CS
float damage = _isNextAutoCrit ? (stats.getCritDamagePct() * stats.AttackDamage.Total) : stats.AttackDamage.Total
```

According to our guidelines though, this should be converted to
```CS
var damage = stats.AttackDamage.Total;
if (_isNextAutoCrit)
{
    damage *= stats.getCritDamagePct()
}
```